src/sections/Meshery/Meshery-integrations/HoneycombGrid.style.js

```diff
@@ -0,0 +1,56 @@
+import styled from "styled-components";
+
+const HoneycombGrid = styled.div`
+  display: flex;
+  --s: 9.375rem;
+  --m: 0.25rem;
+  --f: calc(var(--s) * 1.732 + 4 * var(--m) - 1px);
+
+  .container {
+    font-size: 0;
+  }
+
+  .container div {
+    background: #00d3a9;
+    width: var(--s);
+    margin: var(--m);
+    height: calc(var(--s) * 1.1547);
+    display: inline-block;
+    font-size: initial;
+    clip-path: polygon(0% 25%, 0% 75%, 50% 100%, 100% 75%, 100% 25%, 50% 0%);
+    margin-bottom: calc(var(--m) - var(--s) * 0.2885);
+    :hover {
+      transition: background-color 0.25s ease-in;
+      background: linear-gradient(180deg, #00D3A9 0%, rgba(201, 252, 246, 0.3) 100%);
+      img {
+        filter: brightness(0) invert(1);
```

Comment:
Instead of this, simply use `grayscale(0.8)`. Else, the icons are completely changing to white color which makes them un-recognizable.

Reply:
Added.